### DISCUSSION TITLE
Where's the best place to post New Extension Ideas or Feature ideas?

### DISCUSSION THREAD

**Discussion by BlueDome77** [TurboWarp/extensions#567] [General]
Issues or Discussions?

**Comment by LilyMakesThings** [comment]
I would say issues since people read them more

**Reply by BlueDome77** [reply]
Would it be considered spam if I posted the same thing in both locations?

**Reply by LilyMakesThings** [reply]
Yes, just write it once in issues and leave it at that

**Reply by BlueDome77** [reply]
Should I transfer my old stuff that doesn’t have any replies?

**Reply by LilyMakesThings** [reply]
Yeah, but you don't need to delete any old messages, just leave them as is.

It's only _probably_ considered spam if you post a suggestion in Discussions and Issues at the same time.
